### PR TITLE
Make whole cards on homepage and news list interactive

### PR DESCRIPTION
**Pull request by annanaj**

### CODE REVIEW

File: src/pages/Homepage/Homepage.tsx [hunk 1]
```diff
@@ -46,32 +46,41 @@ export default function NewsList() {
...
-							<div className="mt-auto flex justify-between gap-4 text-sm text-gray-500">
-								<p className="self-end whitespace-nowrap">
-									{formatPublishedDate(
-										newsItem.time_published
-									)}
-								</p>
-								<p>{newsItem.authors}</p>
+							<div className="flex h-full flex-col">
+								<a
+									aria-hidden="true"
```
Comment (crishpeen) [inline src/pages/Homepage/Homepage.tsx]: tbh, I am not 100 % sure if it wouldn't be better if the title link had the before element. Now it is a bit weird, that the screen reader finds an article without any title.

Comment (crishpeen) [reply src/pages/Homepage/Homepage.tsx]: But I understand it is hard if the link underlines.

File: src/pages/Homepage/Homepage.tsx [hunk 2]
```diff
@@ -46,32 +46,41 @@ export default function NewsList() {
...
+									href={`${routes.newsList}`}
```
Comment (crishpeen) [inline src/pages/Homepage/Homepage.tsx]:
```suggestion
									href={routes.newsList}
```

File: src/pages/Homepage/Homepage.tsx [hunk 3]
```diff
@@ -46,32 +46,41 @@ export default function NewsList() {
...
+									onClick={handleClick}
+									className="hover:underline hover:decoration-gray-500"
+								>
+									<h2 className="relative z-10 text-xl font-bold text-gray-900">
```
Comment (crishpeen) [inline src/pages/Homepage/Homepage.tsx]: why z-10?

File: src/pages/NewsList/NewsList.tsx [hunk 4]
```diff
@@ -127,51 +127,63 @@ export default function NewsList() {
 									<a
 										href={`${routes.newsItem}?id=${newsItem.time_published}`}
 										onClick={handleClick(newsItem)}
+										className="before:absolute before:inset-0 hover:underline hover:decoration-gray-500"
```
Comment (crishpeen) [inline src/pages/NewsList/NewsList.tsx]: it looks like you have the same code twice in your codebase, here and in HP, maybe add a component?

File: src/pages/Homepage/Homepage.tsx (hunk 3 above)
Comment (annanaj) [reply src/pages/Homepage/Homepage.tsx]: tailwind default scale, z-0, z-10, z-20,...

**Pull request merged**